### DISCUSSION TITLE
Road to Bezier-PPM 3.0

### DISCUSSION THREAD

**Discussion by germknoedlspeck** [Any2HRTF/PPM#52] [General]
I've opened this issue because I noticed potential lack of detail when manually registering pinnae:

* The cavum conchae of a target can be tilted compared to the cavum conchae in the template, and thus the shape keys "Cavum_conchae-Depth" and "Antitragus-Crease" may not suffice in modelling this specific anatomic region, which is highly relevant to HRTFs. I suggest to add a shape key "Cavum_conchae-Tilt" (or -Angle or -Rotation, for that matter) to cover this.

* The position of the ear canal is not covered by any parameter, and would yield potentially high geometric local errors. I'm not sure about the importance of this issue, but maybe we can have a look at it once we've done some registrations.

I'm inviting @eachtling @felixperfler and @woegerbauerJ to comment similar requests during your manual registration processes :)

**Comment by germknoedlspeck** [comment]
* Insicura intertragica: increase influence by Tragus-Bendy and Antitragus-Bendy

**Comment by eachtling** [comment]
* Mesh-continuity issues when changing scales of, for example, Helix_middle-Bendy and Helix_low-Bendy to high values (>1.5).

**Comment by germknoedlspeck** [comment]
* introduce boundary condition for the surface not to intersect (maybe specify material or modifier for Mesh object?)

**Comment by germknoedlspeck** [comment]
* intersect edge outer boundary with ellipse to ensure smooth fringe

**Comment by germknoedlspeck** [comment]
> * introduce boundary condition for the surface not to intersect (maybe specify material or modifier for Mesh object?)

https://blender.stackexchange.com/questions/73869/how-to-stop-self-intersection-of-model ?

**Reply by germknoedlspeck** [reply]
The boundary condition has to prevent faces with opposite-facing normals to intersect

**Comment by germknoedlspeck** [comment]
* display floor with higher resolution (control panel, display, grid floor)
